apps/openmw/mwmechanics/actors.cpp

```diff
@@ -845,10 +834,7 @@ namespace MWMechanics
             }
         }
         else
-        {
-            static const float fHoldBreathTime = world->getStore().get<ESM::GameSetting>().find("fHoldBreathTime")->getFloat();
-            stats.setTimeToStartDrowning(fHoldBreathTime);
-        }
+            stats.setTimeToStartDrowning(world->getStore().get<ESM::GameSetting>().find("fHoldBreathTime")->getFloat());
```

Comment:
Can we keep the ```static const float``` variable for fHoldBreathTime? I don't think we want to retrieve this GMST for every actor, every frame.